lib/model/CMetricBucketGatherer.cc

```diff
@@ -1057,7 +1057,6 @@ bool CMetricBucketGatherer::acceptRestoreTraverserInternal(core::CStateRestoreTr
         CRestoreFeatureData<model_t::E_MultivariateMin> restore;
         if (restore(traverser, dimension, isCurrentVersion, *this, m_FeatureData) == false) {
             LOG_ERROR(<< "Invalid multivariate min data in " << traverser.value());
-            return false;
```

Comment:
I think this line should be reverted too since it's the only change in this file now.